src/Void.Core/CommandLanguage.fs

```diff
@@ -15,17 +15,20 @@ module CommandLanguage =
     let handleRequest language request =

```

Comment:
Because before I was only exposing subscription stuff on it, but now I am actually exposing all the bus-like stuff (e.g. `publish`).